include/mtrie.h

```diff
@@ -0,0 +1,40 @@
+// Sprint 2 Mock Trie Implementation
+// Ruolin Zheng
+
+
+// minimal C implementation of a mock trie
+
+#ifndef __TRIE_H__
+#define __TRIE_H__
```

Comment:
The variable naming scheme here is not standardized across source files. Please use `_FULLTEXT_NAME_H_` across the board (that is, this would be `_FULLTEXT_TRIE_H_`).